docs/team/weiping.adoc

```diff
@@ -0,0 +1,45 @@
+= Khoo Wei Ping - Project Portfolio
+ifdef::env-github,env-browser[:outfilesuffix: .adoc]
+:imagesDir: ../images
+:stylesDir: ../stylesheets
+
+== Project: Codii
+[https://github.com/CS2103AUG2017-T17-B1/main[Codii]] is a desktop address book application targeted at debt collectors. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 6kLoC.
+
+Codii is enhanced from [https://github.com/nus-cs2103-AY1718S1/addressbook-level4[AddressBook - Level 4]] which is a desktop address book application used for teaching Software Engineering principles.
+
+*Code contributed*: [https://github.com/CS2103AUG2017-T17-B1/main/tree/master/collated/main/khooroko.md[Functional code]] [https://github.com/CS2103AUG2017-T17-B1/main/tree/master/collated/test/khooroko.md[Test code]] {give links to collated code files}
```

Comment:
Not sure how to make these links relative instead of giving full url

Reply:
can take a look at this link here: https://github.com/blog/1395-relative-links-in-markup-files